mantidproject/mantid
# Add release note templates for future releases

**Description of work.**
These are the new templates that will be used to generate the release notes for the next release. Previously the templates for the separate release notes were hard coded into release.py. However the changes to the release note process mean that this is no longer a suitable method. By having separate templates this will make it easier if changes need to be made in future.

The conf.py file has been updated to exclude the templates from the toc_tree as otherwise it is likely to cause build failures (see #33599)

**To test:**

- check that the headings for all except Muons in the templates match those listed in the Release Note Guide (https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)
- check that the headings for Muons match those listed in the revised file structure for Muons (see #33672 for details)
- check that each heading has an amalgamate sphinx directive with it.
- There should be no developer warnings or release note version information in the templates

Part of the release note project (#33593)

*This does not require release notes* because **this is not a user facing part of release notes**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
